httomo_backends/scripts/json_pipelines_generator.py

```diff
@@ -22,7 +22,7 @@
 # Directory containing YAML files relative to this script
 PIPELINES_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..", "pipelines_full")
 
-ignored_methods = ["calculate_stats"]
+ignored_methods = ["standard_tomo_diad","standard_tomo","calculate_stats"]
```

Comment:
Actually there is no more such thing as `standard_tomo_diad` loader so you can remove it.  Do you still need to ignore `standard_tomo` ?

Reply:
yes, cause my front-end generates `standard_tomo` already, both `standard_tomo` and `calculate_stats` are methods that have a different pattern compared to the rest of the methods, if I don't ignore it here the final config will have a duplicated `standard_tomo` method

Reply:
I see... well you can then do `ignored_methods = ["standard_tomo","calculate_stats"]`